client/src/App.js

```diff
@@ -20,6 +21,9 @@ const App = () => {
         </div>
       )}
       <LoginButton />
+      {
+       user && <UserInfo />
+     }
```

Comment:
This part of the profile will be shown to other users whether it's the user profile or not.